src/graph.cpp

```diff
@@ -261,7 +260,7 @@ std::shared_ptr<Edge> Graph::findEdge(const unsigned int &v1id, const unsigned i
 
 /**
  * A backtracking function for getting the value of the smallest weighted path for the given graph
- * Time Complexity: O(2^N) (worst case)
+ * Time Complexity: O(N!) (worst case)
  * @param dists - Distance map for the graph used
```

Comment:
I also think, as mentioned previously, this parameter here is unnecessary when you can just access the distance Matrix directly

Reply:
This function doesn't have parameters anymore, please update the documentation